### PR TITLE
feat: registry v2

### PR DESCRIPTION
**Pull request by lucacasonato**

### CODE REVIEW

File: public/posts/registry2.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+Since the beginning the goal for deno.land/x is to give all modules in the Deno
```
Comment (bartlomieju) [inline public/posts/registry2.md:1]: `is` should be changed to `was` or `has been`?

File: public/posts/registry2.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+ecosystem a reliable, easy to use, and free host. With the increased adoption of
```
Comment (bartlomieju) [inline public/posts/registry2.md:2]: `s/host/hosting`

File: public/posts/registry2.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+- removing non versioned imports
+- and removing the GitHub API rate limits on the website
+
+To do this we setteled on a design where all source code is stored our servers,
+and we retrieve a new version from GitHub whenever it is published. We get
+notified of this by a GitHub Webhook you add to your repository.
+
+## What has changed?
+
+- Source code is now not stored on raw.githubusercontent.com, rather on our own
```
Comment (bartlomieju) [inline public/posts/registry2.md:26]: `Source code is no longer stored on raw.githubusercontent.com but rather on our own servers`

File: public/posts/registry2.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  servers.
+- Publishing modules works through a GitHub Webhook now, rather than by opening
+  a PR on the deno_website2 repository.
+- You can not import from arbitrary commits or branches anymore, only tags /
+  releases. No more `https://deno.land/std@master`.
```
Comment (bartlomieju) [inline public/posts/registry2.md:31]: No more `https://deno.land/std@master` is ambiguous... Also one can't import explicitly `https://deno.land/std@master` anymore? That's surprising

File: public/posts/registry2.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+## Future plans
+
+With this architechture we have the possibility to add all kinds of features.
+Here are a few that we have planned:
+
+1. Display download counts for all modules
+2. Give all modules a score based on how well their module is maintained
+   - do `deno lint`, `deno fmt`, `deno doc`, and the TypeScript compiler report
+     any issues
+   - are imports version locked
```
Comment (bartlomieju) [inline public/posts/registry2.md:62]: Are dependencies pinned to specific version?

File: public/posts/registry2.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+a release.
+
+## Future plans
+
+With this architechture we have the possibility to add all kinds of features.
+Here are a few that we have planned:
+
+1. Display download counts for all modules
+2. Give all modules a score based on how well their module is maintained
+   - do `deno lint`, `deno fmt`, `deno doc`, and the TypeScript compiler report
```
Comment (bartlomieju) [inline public/posts/registry2.md:60]: Verify that `deno lint`, `deno fmt`, `deno doc` and type checking produce no errors

File: public/posts/registry2.md (hunk 4 above)
Comment (lucacasonato) [reply public/posts/registry2.md:31]: > Also one can't import explicitly https://deno.land/std@master anymore? That's surprising

We cant upload all commits for all repos to S3, so we have to remove this feature.

File: pages/index.tsx [hunk 7]
```diff
@@ -1,44 +1,46 @@
...
-    );
-  }, []);
+  // const [thirdPartySelection, setThirdPartySelection] = useState<
+  //   SimpleEntry[] | null
+  // >(null);
+  // useEffect(() => {
+  //   setThirdPartySelection(
+  //     RandomEntriesFromArray(thirdPartyEntryPool, NUM_THIRD_PARTY)
+  //   );
+  // }, []);
```
Comment (ry) [inline pages/index.tsx:43]: Consider removing dead code

File: pages/index.tsx [hunk 8]
```diff
@@ -351,43 +353,43 @@ const InstallSection = () => {
...
+//   for (let i = 0; i < Math.min(count, source.length); i++) {
+//     const s = Math.floor(sourceArray.length * Math.random());
+//     selection.push(sourceArray[s]);
+//     source.splice(s, 1);
+//   }
 
-  return selection;
-};
+//   return selection;
+// };
```
Comment (ry) [inline pages/index.tsx:367]: Remove dead code?

File: public/posts/registry2.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
+The goal of deno.land/x has been to provide a central location for third party
+Deno modules consistant with how Deno operates. We want people to be able to
```
Comment (bartlomieju) [inline public/posts/registry2.md:2]: `s/consistant/consistent`

File: public/posts/registry2.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+2. Give all modules a score based on how well their module is maintained
+   - verify that `deno lint`, `deno fmt`, `deno doc` and type checking produce
+     no errors
+   - are dependencies pinned to specific version
+   - does the module have a LICENCE, README.md
+3. Display dependencies of modules on the site
+4. Serve a JS (type stripped) version of all TypeScript files in a module, to be
+   imported directly from a web browser
+
+If you have any comments or feedback, please let open an issue on the
```
Comment (bartlomieju) [inline public/posts/registry2.md:64]: remove `let`?

File: pages/index.tsx (hunk 7 above)
Comment (lucacasonato) [reply pages/index.tsx:43]: I wanna re-add this in the future, but I can remove it for now.

**Pull request merged**